DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py

```diff
@@ -94,27 +97,34 @@
 	process.castorDigis.InputLabel = rawTag
 
 process.emulTPDigis = process.simHcalTriggerPrimitiveDigis.clone(
-   inputLabel = ["hcalDigis", 'hcalDigis'],
+   inputLabel = cms.VInputTag("hcalDigis", "hcalDigis:ZDC"),
    FrontEndFormatError = True,
    FG_threshold = 2,
    InputTagFEDRaw = rawTag,
    upgradeHF = True,
    upgradeHE = True,
    upgradeHB = True,
-   inputUpgradeLabel = ["hcalDigis", "hcalDigis"],
+   inputUpgradeLabel = cms.VInputTag("hcalDigis", "hcalDigis:ZDC"),
    # Enable ZS on emulated TPs, to match what is done in data
    RunZS = True,
    ZS_threshold = 0
 )
 
+#inserting zdc emulator after tp digis
+process.etSumZdcProducer = cms.EDProducer('L1TZDCProducer',
+                                          hcalTPDigis = cms.InputTag("emulTPDigis"),
+                                          bxFirst = cms.int32(-2),
+                                          bxLast = cms.int32(3)
+)
+
 process.hcalDigis.InputLabel = rawTag
 process.emulTPDigisNoTDCCut = process.emulTPDigis.clone(
      parameters = cms.untracked.PSet(
 	ADCThresholdHF = cms.uint32(255),
 	TDCMaskHF = cms.uint64(0xFFFFFFFFFFFFFFFF)
      )
 )
-process.HcalTPGCoderULUT.LUTGenerationMode = False
+process.HcalTPGCoderULUT.LUTGenerationMode = True
```

Comment:
is this ultimately really needed given the answer at https://github.com/cms-sw/cmssw/pull/46407/files#r1804677477 ?

Reply:
This should be reverted to false, ZDC channels were now added to HcalL1TriggerObjects, fixing the need to set this to true.

Reply:
I can make the reversion to False requested, but I would like to also understand the failed check first and fix everything in one go - if I click through on the error details I can't actually navigate the page after to find anything. Any guidance on where to look to fix this failed check is appreciated

Reply:
> I would like to also understand the failed check first and fix everything in one go

nothing failed, these were transient das errors.

Reply:
Ah! Apologies, I did not understand that - I will revert to False

Reply:
Reverted to false in latest commit